ci/scripts/clone-build_ci.sh

```diff
@@ -49,21 +44,21 @@ done
 # start output file
 {
  echo "Automated global-workflow Testing Results:"
- echo "Machine: ${CI_HOST}"
+ echo "Machine: ${MACHINE_ID^}"
  echo '```'
  echo "Start: $(date) on $(hostname)" || true
  echo "---------------------------------------------------"
 }  >> "${outfile}"
 ######################################################################
 
-cd "${repodir}"
+cd "${repodir}" || exit
 # clone copy of repo
 if [[ -d global-workflow ]]; then
   rm -Rf global-workflow
 fi
 
-git clone "${repo_url}"
-cd global-workflow
+git clone "${REPO_URL}"
+cd global-workflow || exit
```

Comment:
`exit` should be `non-zero` in case of a failure.
```suggestion
cd global-workflow || exit 1
```

Reply:
Yes!